sycl/source/detail/error_handling/enqueue_kernel.cpp

```diff
@@ -177,7 +178,6 @@ bool handleInvalidWorkGroupSize(const device_impl &DeviceImpl, pi_kernel Kernel,
 
   const plugin &Plugin = DeviceImpl.getPlugin();
   RT::PiDevice Device = DeviceImpl.getHandleRef();
-
```

Comment:
Was this change done intentionally? If no, I suggest to revert it. The same applies to a new line above (169)